scipy/optimize/lbfgsb.py

```diff
@@ -275,6 +275,7 @@ def _minimize_lbfgsb(fun, x0, args=(), jac=None, bounds=None,
             iprint = disp
 
     n_function_evals, fun = wrap_function(fun, ())
+    n_jac_evals, jac = wrap_function(jac, ())
```

Comment:
Is it ever possible for `n_jac_evals[0] != n_function_evals[0]`? Your description suggests that they should match.

Reply:
agreed that direct tracking is the most robust, no good reason not to.